src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -863,7 +863,7 @@ private function loadDeleteForm(): void
         $this->template->assign('deleteForm', $deleteForm->createView());
     }
 
-    private function getHiddenJsonField(string $name, string $json): SpoonFormHidden
+    private function getHiddenJsonField(string $name, ?string $json = null): SpoonFormHidden
```

Comment:
But why? The `= null` is just there to provide some extra information. It doesn't matter it's the default and the developer should 'know how it works'. If that's the way you think about it, I'll make a PR removing all `public` keywords from functions since it's the default and developers should just know that. Sounds like some visual debt bs to me